src/Tribe/API.php

```diff
@@ -270,6 +270,40 @@ public static function update_event_cost( $event_id ) {
 		}
 
 		/**
+		 * Update user-created Additional Fields
+		 *
+		 * @param $event_id
+		 * @param $data
+		 */
+		public static function update_additional_fields ( $event_id, $data ) {
+			$additionalFieldTags = array_map(
+				function( $field ) { return $field['name']; },
```

Comment:
We cannot use Anonymous because it is not Supported by PHP 5.2, a good solution in this case is to use [`wp_list_pluck`](https://codex.wordpress.org/Function_Reference/wp_list_pluck)